stubs/psutil/psutil/__init__.pyi

```diff
@@ -223,6 +223,7 @@ class Process:
     def terminate(self) -> None: ...
     def kill(self) -> None: ...
     def wait(self, timeout: float | None = None) -> int: ...
+    def net_connections(self, kind: str = "inet") -> list[pconn]: ...
```

Comment:
We can also add `@deprecated` to the `connections` method (doesn't have to be in this PR though).

Reply:
Ok, I will in the next one.

Reply:
A bit later, maybe tomorrow.